modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,668 @@
+--
+.Input
+[source,bash]
+----
+rpk topic list
+----
+
+.Output
+[source,bash]
+----
+NAME   PARTITIONS  REPLICAS
+bad    1           1
+good   1           3
+----
+--
+
+See also:
+xref:upgrade:migrate/data-migration.adoc#change-topic-replication-factor[Change topic replication factor]
+
+[[maintenance-mode]]
+=== No brokers in maintenance mode
+
+Check to ensure that no brokers are in maintenance mode.
+
+[.side-by-side]
+--
+.Input
+[source,bash]
+----
+rpk cluster maintenance status
+----
+
+.Output
+[source,bash]
+----
+NODE-ID  ENABLED  FINISHED  ERRORS  PARTITIONS  ELIGIBLE  TRANSFERRING  FAILED
+1        false    -         -       -           -         -             -
+2        false    -         -       -           -         -             -
+3        false    -         -       -           -         -             -
+----
+--
+
+See also:
+xref:manage:node-management.adoc#place-a-broker-in-maintenance-mode[Remove a broker from maintenance mode]
+
+[[decom-state]]
+=== No brokers in decommissioned state
+
+Check to ensure that no brokers are in a decommissioned state.
+
+[.side-by-side]
+--
+.Input
+[source,bash]
+---- 
+rpk redpanda admin brokers list
+----
+
+.Output
+[source,bash]
+---- 
+NODE-ID  NUM-CORES  MEMBERSHIP-STATUS  IS-ALIVE  BROKER-VERSION
+0        1          active             true      v24.1.6 - 5e880f6fd1a610d0991b00e32c012a03b14888ca
+1        1          active             true      v24.1.6 - 5e880f6fd1a610d0991b00e32c012a03b14888ca
+2        1          active             true      v24.1.6 - 5e880f6fd1a610d0991b00e32c012a03b14888ca
+----
+--
+
+See also:
+* xref:manage:cluster-maintenance/decommission-brokers.adoc[Decommission Brokers]
+
+== Level 2 production readiness
+
+The Level 2 readiness checklist helps you to confirm that you can monitor and support your environment on a sustained basis, and includes the following checks:
+
+- You have adhered to 2-day operations best practices
+- You have sufficient ability to diagnose and recover from any issues that arise or failures that occur
+
+=== Environment configuration
+
+Confirm that you have a xref:deploy:deployment-option/self-hosted/manual/production/dev-deployment.adoc[development environment] or test environment configured so that you can evaluate upgrades and new versions before rolling them straight to production.
+
+=== Monitoring
+
+Confirm that xref:manage:monitoring.adoc[monitoring] is configured with xref:manage:monitoring.adoc#configure-prometheus[Prometheus], xref:manage:monitoring.adoc#generate-grafana-dashboard[Grafana], or https://www.datadoghq.com/product/log-management/[Datadog^] to scrape metrics from all Redpanda brokers at a regular interval.
+
+
+=== System log retention
+
+Confirm that system logs are being captured and stored for an appropriate period of time (minimally, 7 days). On bare metal, this may be journald, however on Kubernetes you may need to have fluentd or equivalent configured and logs sent to a central location.
```

Comment:
```suggestion
Check that system logs are being captured and stored for an appropriate period of time (minimally, 7 days). On bare metal, this may be journald. On Kubernetes you may need to have fluentd or an equivalent configured, with logs sent to a central location.
```